### ISSUE TITLE
Additions to content pages from RAD FAQs

### ISSUE THREAD

**Issue by llienfec** [18F/fec-cms#1705]
Received the following requests for updates and changes to content pages based on information currently provided in RAD FAQs (helping with feature parity).  

- [x] The transition site FAQs include a question about whether a committee should file 48-hour notices late if it missed the 48 hour window.  Could we include the following sentence from the FAQs in our 48-hour notices reporting example on the new website: “48-Hour Notices must be filed within 48 hours of a contribution’s receipt.  Should you miss this deadline, we still recommend that you file the notice, provided it is before the date of the applicable election.”
- [x] The RFAI page of the new website (https://www.fec.gov/help-candidates-and-committees/request-additional-information/) has a great description, but could we consider also including a notated image of a sample RFAI like the one found at https://transition.fec.gov/rad/FederalElectionCommission-RAD-RespondingtoRFAIs.shtml?
- [x] On the RFAI page of the new website, there is a legal citation about Best Efforts at the bottom.  However, Best Efforts is not explained on that page.  Could we include a link to the page where the Best Efforts information is provided?  I think it’s provided at https://www.fec.gov/help-candidates-and-committees/keeping-records/records-receipts/, but the Best Efforts information appears in the middle of that page – could we also consider creating a separate link for it, so the user can jump to that information rather than needing to scroll?
- [x] The RAD Process document, to which the RFAI page on the new website provides a link, needs to be updated to include the Reports Processing Branch.  We’ll be glad to help with this if needed.
- [x] The new website does not seem to have a section with tips on amending reports.  Could we include something on the “Filing Reports” page (https://www.fec.gov/help-candidates-and-committees/filing-reports/) – perhaps insert “Amending Reports” under “Troubleshooting,” or create a separate small subsection above “Troubleshooting”?  Here are the FAQs we’d like to duplicate (in a narrative format rather than question & answer) for the new website: “I need to amend a report; do I need to file all subsequent reports?” and “I received an RFAI referencing a report filed with the paper FEC forms.  However I now file electronically.  How do I amend?” (please see https://transition.fec.gov/rad/FederalElectionCommission-RAD-RespondingtoRFAIs.shtml#amendreport and https://transition.fec.gov/rad/FederalElectionCommission-RAD-RespondingtoRFAIs.shtml#paperamendment).

**Comment by AmyKort** [comment]
Looks good to me. Looping in @gsmaragdis and @dorothyyeager

**Comment by gsmaragdis** [comment]
Fine with me.  Dorothy is out today, but I am sure she will give us her thoughts when she gets back.

**Comment by dorothyyeager** [comment]
I like all of these! I had wondered about no amendment information so that's a good idea in particular.

**Comment by dorothyyeager** [comment]
One more thought - we need to come up with something for the forthcoming electioneering communications page (part of the new corporation/ssf pages). Right now the page links to the RAD FAQ on the issue. www.fec.gov/rad/other_filers/FederalElectionCommission-RAD-OtherFilers.shtml

**Comment by llienfec** [comment]
Here are the updates thus far:

### Done
**48-hr notice language added to 1st paragraph of reporting example:**
If the committee misses this deadline, the Commission still recommends filing the notice, provided it is filed before the date of the applicable election.

**RFAI page & best efforts references**
Removed the legal cite at bottom b/c it's not addressed within context of page.  We're not able to link directly to the best efforts section of the page requested above due to limitations in Wagtail's templates and the inability to create anchor links.  Instead, added "Recording receipts & best efforts to document receipts" to the "read next" section, so that users will know to expect other info on the best efforts page.

### To-do
- Ask Jim to create an RFAI image with annotations, updated with the website's image styling
- Created an updated RAD process document.  Currently getting it approved.
- Working on draft language for amendment section.

**Comment by llienfec** [comment]
Drafted language for the new amendment page using the campaign guides, F1 and F2 reporting examples and efiling amendment webpage section.

The new page will live here for each committee type:
H4CC > [Committee type] > Filing reports > Troubleshooting 

Dorothy, Amy and Rebecca reviewed the new page.  It has been added to wagtail, published, and added to the candidate, SSF, nonconnected and party committee pages.

### To-do
This issue is currently blocked while we wait for a couple of things to happen:
- Jim will help with updated RFAI image when he is in the office next week.
- RAD process document is with RAD manager for review.

**Comment by llienfec** [comment]
The great @jenniferthibault is going to help us out with an RFAI image that matches our new styling while Jim is out of office.

**Comment by jenniferthibault** [comment]
@llienfec I wasn't exactly sure where in the page you are thinking of placing this, so for mockup purposes I just stuck it at the end, knowing this was likely wrong, and you'd put it where it belongs :)

Using the existing page template to do this, we have a couple of options:

1. We can place the text inside the image, just like the old one. This keeps everything together in one, with the call-outs right next to the example. This is done using just the image block field in Wagtail.

![screen shot 2018-01-31 at 3 32 47 pm](https://user-images.githubusercontent.com/11636908/35646222-7ea5ea78-069c-11e8-8d4c-1c240976bd99.png)

2.  We can highlight the image, but place the text annotations underneath it using an image block followed by a paragraph block. This is a smidge better for screenreaders, since it keeps the text as live and readable, instead of as an image. (a very minor improvement, since the whole RFAI image is text, and since the info here is repeated in the surrounding paragraphs anyways).

![screen shot 2018-01-31 at 3 33 14 pm](https://user-images.githubusercontent.com/11636908/35646349-eaeaaee4-069c-11e8-8667-d57ba543d977.png)


Thoughts? (cc @JonellaCulmer if this is of interest to see!)

**Comment by JonellaCulmer** [comment]
Thanks for the cc, @jenniferthibault. Just my two cents. I think the way we do this should mirror the way the Reporting examples are implemented, with the image to the left and live annotations to the right. Or are you suggesting the text be placed below the image to keep the RFAI image large?

Example: https://www.fec.gov/help-candidates-and-committees/filing-reports/political-party-contributions/

**Comment by jenniferthibault** [comment]
You are exactly right, I totally agree, that would be the ideal layout. I should have acknowledged that first before skipping to the workarounds. Thank you so much for the prompt here!

The immediate hiccup here is that these two pages are built on different wagtail page templates, and those two templates have different content blocks available in them.

The [RFAI page](https://www.fec.gov/help-candidates-and-committees/request-additional-information/) is built using the Custom page template. Here are the content blocks available to insert into the body of the custom page template:
<img width="1080" alt="screen shot 2018-01-31 at 4 39 58 pm" src="https://user-images.githubusercontent.com/11636908/35648943-6d2a3264-06a5-11e8-97e1-e0c598e44be6.png">

The "How to report" pages, including the [How to report political party contributions](https://www.fec.gov/help-candidates-and-committees/filing-reports/political-party-contributions/) is built on a different template called the Reporting example page template. Here are the content blocks available to insert into the body of the custom page template:
<img width="1035" alt="screen shot 2018-01-31 at 4 41 25 pm" src="https://user-images.githubusercontent.com/11636908/35649044-c8c4ead8-06a5-11e8-9548-18f84d45151b.png">

So in order to get that format in the Custom page template and use it on the RFAI page, we'd need to make changes to the Custom page template in Wagtail. It's possible! Very possible! How we've approached this decision before is to look at if this is a one-off scenario, or if we're likely to want this functionality on other instances of the Custom page template. If the answer is yes here, we may want to make that a follow-up task so that it doesn't block the info from being available using the tools we have sooner. 

A next step I can see for me is to find a way to share what I know about template components with other folks on the team, this will be really helpful for you @JonellaCulmer and I think for content managers!

**Comment by llienfec** [comment]
Both of these images are so much improved from the original.  I agree that the ideal would be to do it like the reporting images to keep design similar across the site where we can.  I think the challenge there (aside from not currently having the capability) is that the image annotations would still be clumped together.  We combat this in the reporting examples by splitting images when we need to talk about different parts of a page image.  Is it worth adding the template?  I think that's a bigger convo re: time & priorities and could be a good discussion for next week.  I'm fine w/ using a work around, especially for now.

Right now, my preference is for # 1.  It's much clearer that the text and the highlighting relate.  If the difference in image size between the two makes a big difference, then I'll relent.

A related thought about image sizes:  When we conducted reporting example usability testing at the DC conference, one of the comments I heard a few different times from users was that they wanted the ability to enlarge the reporting example images.  It also may be an interesting convo for next week.

**Comment by dorothyyeager** [comment]
This is going to be super useful. I saw Jen's note about the custom template in Wagtail and can say, yes, for sure, there are definitely more cases out there for having some more types of content blocks available to us, especially for those pages that are longer than we'd like or have tables.

**Comment by JonellaCulmer** [comment]
Thanks for all of that @jenniferthibault. You answered my final question by explaining the wagtail options available to us. 

If I have a vote, I say option 2. That would make sure we're maintaining that live and readable text as we do with the reporting examples. It's not ideal but at least we're maintaining accessibility. And then definitely pursue modifying the template so that these example-type pages all mirror one another. (Maybe we need a Wagtail epic so that we can also talk about tables. Hmm....)

Also, I would love to have a discussion with you about the template components. Either in one of our one-on-ones, design sync (if there's nothing more pressing to discuss), or at PI planning (if there's time).

**Comment by llienfec** [comment]
If y'all think #2 is best, that's fine with me.

**Comment by jenniferthibault** [comment]
@llienfec it will only take me a second to finalize and export the image— are there any changes to the text you'd like to make before that happens?

**Comment by llienfec** [comment]
@jenniferthibault   The image looks great.  

I was debating about adding highlighting to the local DC number because it's a direct line to the analyst.  But after looking at it again, I think editing the last bullet to include that information would be effective.

**Comment by llienfec** [comment]
Issue is unblocked - Created an updated RAD Review Procedures Overview document, which was approved to post by RAD management.  Coordinated with @dorothyyeager to figure out where to save it and learned how to update PDF metadata.  Now the title is showing up correctly in Chrome.

Also uploaded and added content re: RFAI page to 1) show image of RFAI using current website styling 2) added bullets to describe highlighting per @jenniferthibault and @JonellaCulmer discussion above 3) updated the contacts language an link to go to the RAD form and contact lookup

**Comment by llienfec** [comment]
Issue is finished.  Quick summary:

- Posted additional 48 hr notice language on reporting example
- Posted updated RFAI image & more language on RFAI page
- Changed confusing best efforts cite on RFAI page to a read next link that goes to BE website page
- Created/approved/posted updated RAD Review Process doc
- Created/approved/posted new amendment page
